### PR TITLE
Named anon_inode io_uring usage

### PR DESCRIPTION
**Pull request by zpytela**

### CODE REVIEW

**Comment by zpytela** [comment]
@WOnder93 plese review, this is a follow-up to https://github.com/fedora-selinux/selinux-policy/pull/1351/commits/ecb0ee0e5

File: policy/modules/kernel/kernel.if [hunk 1]
```diff
@@ -4619,3 +4619,20 @@ interface(`kernel_io_uring_use',`
...
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`kernel_io_uring_rw',`
+	gen_require(`
+		type io_uring_t;
+	')
+	allow $1 io_uring_t:anon_inode common_anon_inode_perms;
```
Comment (WOnder93) [inline policy/modules/kernel/kernel.if]: After 65b9e0bdceb7e6adbe308f9a591b103cba6986ef, we shouldn't use `common_anon_inode_perms` any more (in favor of listing the permissions verbatim in the few relevant interfaces). In fact, the macro should have been deprected in that commit, but I didn't think of that.

Any io_uring fd use should be covered by `kernel_io_uring_use()` already and it allows most of the perms from `common_anon_inode_perms` except `ioctl` (which I think shouldn't be needed, but I'm not sure). Is this motivated by seeing an actual `ioctl` AVC?

Comment (zpytela) [reply policy/modules/kernel/kernel.if]: These permissions were needed:
(allow named_t io_uring_t (anon_inode (create map read write)))
(allow ndc_t io_uring_t (anon_inode (create map read write)))

Comment (zpytela) [reply policy/modules/kernel/kernel.if]: You are right, I completely overlooked the actual content of the existing interface.

**Review by WOnder93 — APPROVED** [review]
This version looks good :+1:

**Pull request merged**